lib/handlers/puback.js

```diff
@@ -3,6 +3,7 @@
 function handlePuback (client, packet, done) {
   var persistence = client.broker.persistence
   persistence.outgoingClearMessageId(client, packet, done)
+  client.broker.emit('deliver', packet, client)
```

Comment:
I think 'ack' is the best of all, actually. I'll get that updated asap!
Il giorno dom 3 apr 2016 alle 18:14 Behrad <notifications@github.com> ha
scritto:

> In lib/handlers/puback.js
> <https://github.com/mcollina/aedes/pull/39#discussion_r58312605>:
>
> > @@ -3,6 +3,7 @@
> >  function handlePuback (client, packet, done) {
> >    var persistence = client.broker.persistence
> >    persistence.outgoingClearMessageId(client, packet, done)
> > +  client.broker.emit('deliver', packet, client)
>
> I believe delivered or ack could be a better name, in Mosca we are using
> delivered @mcollina <https://github.com/mcollina>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mcollina/aedes/pull/39/files/7897baca0c9db516cd179f3d64497c9527b84f5a#r58312605>
>